cookbooks/aws-parallelcluster-install/resources/efs/partial/_install_efs_utils_debian.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :install_utils do
+  package_name = "amazon-efs-utils"
+  efs_utils_tarball = node['cluster']['efs_utils']['tarball_path']
+
+  # Do not install efs-utils if a same or newer version is already installed.
+  return if Gem::Version.new(get_package_version(package_name)) >= Gem::Version.new(node['cluster']['efs_utils']['version'])
+
+  # On Ubuntu, amazon-efs-utils and stunnel are installed from source
+  # Because their OS repos do not have amazon-efs-utils and new stunnel
+  # Get EFS Utils tarball
+  remote_file efs_utils_tarball do
+    source node['cluster']['efs_utils']['url']
+    mode '0644'
+    retries 3
+    retry_delay 5
+    not_if { ::File.exist?(efs_utils_tarball) }
+  end
+
+  # Verify tarball
+  ruby_block "verify EFS Utils checksum" do
+    block do
+      require 'digest'
+      checksum = Digest::SHA256.file(efs_utils_tarball).hexdigest
```

Comment:
Thought: we might want to simplify this part, because `remote_file` resource has `checksum` attribute which should be possible to use in order to validate the file. However it needs a double check because the documentation is not clear.